recipes-core/coreutils/coreutils_8.%.bbappend

```diff
@@ -11,14 +11,16 @@ DEPENDS += "shadow-native pseudo-native niacctbase"
 RDEPENDS_${PN}-hostname += "niacctbase"
 
 # preserve existing behavior when installing base package
-RDEPENDS_${PN}_class-target += "\
```

Comment:
I think the way you're supposed to fix this error is by using something like
```bash
RDEPENDS_${PN}_class-target_append += ...
```
... or something like that. I might be mistaken, but I would like to preserve the `${PN}` semantic, if we can.

Reply:
Tried that, but it didn't work. Take a look at the final comment from Richard Purdie in this [thread](https://patchwork.openembedded.org/patch/98957/) that might explain why.

Reply:
Ah; that's slightly annoying. Could you add a section to the bitbake wiki document with a link to that thread. I imagine this will come up many more times. :/